### DISCUSSION TITLE
4-relay shutter modes (2 blinds in the same room)

### DISCUSSION THREAD

**Discussion by fabiorcosta** [arendst/Tasmota#16854] [General]
I tried to setup this board as follow:
SetOption80 1 
ShutterRelay1 1 
interlock 1,2
ShutterRelay2 1
interlock 1,2 3,4

but I cant have relays 3 and 4 interlocked.

**Comment by barbudor** [comment]
This might be hardware related
What gpios are you using?
Was that not occurring with the original firmware?

**Reply by fabiorcosta** [reply]
@barbudor The hardware I am using is the photo. and GPIOs designated by them 
![IMG_2404](https://user-images.githubusercontent.com/82267134/196436160-3ff45e7c-a593-4059-b10b-2aa4c91ec7d8.jpg)

![image](https://user-images.githubusercontent.com/82267134/196436198-3f830b5c-3f2d-4908-bbde-2e4da4ceec34.png)

**Reply by sfromis** [reply]
Bad hardware setup, using gpio16 for a relay is not workable if you do not want activity at boot.

**Comment by fabiorcosta** [comment]
Solved changing the GPIO. Thanks.
